packages/api/src/middleware/expoJudgeAccessMiddleware.ts

```diff
@@ -0,0 +1,43 @@
+import { ExpoJudgingSession, ExpoJudgingSessionContext } from '@hangar/database';
+import { NextFunction, Request, Response } from 'express';
+import { logger } from '../utils/logger';
+
+export const expoJudgeAccessMiddleware = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  const {
+    judge,
+    entityManager: em,
+    params: { id: ejsId },
+  } = req;
+
+  try {
+    const ejsCount = await em.count(ExpoJudgingSession, { id: ejsId as string });
+    if (ejsCount < 0) {
```

Comment:
When will this ever be true?

Reply:
you're idk how i didn't catch that more importantly i don't know why i did that. but fixed it.